### PR TITLE
Release TrueLearn 1.1.0

### PR DESCRIPTION
**Pull request by KD-7**
### Release Checklist

- [x] Update version no.
- [x] Update docs
- [x] Populate changelog
- [x] Get changes from main
- [x] Remove wordplotter warning

### CODE REVIEW

**Comment by yuxqiu** [comment]
We can also remove the warning in WordPlotter Example.

**Comment by KD-7** [comment]
Yep, I had done so locally, just hadn't pushed yet

> ~We can also remove the warning in WordPlotter Example.~ I've removed it.

File: docs/changelog.rst [hunk 1]
```diff
@@ -3,5 +3,43 @@ Change Log
...
+**TrueLearn 1.1.0 (13-08-2023)**
+
+*Added*
+
+- The option to visualise the variance in Bar, Dot, Line plotters. By default this is
+  enabled except in the Line plotter.
+
+*Changed*
+
+- INK classifier now uses a binary skill representation.
```
Comment (yuxqiu) [inline docs/changelog.rst:23]: Should be Interest Classifier

**Review by yuxqiu — CHANGES_REQUESTED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (KD-7) [reply docs/changelog.rst]: Fixed

**Review by yuxqiu — APPROVED** [review]

**Pull request merged**